src/components/elements/tableFilters/filters/FilterButton.tsx

```diff
@@ -1,18 +1,35 @@
-import { Button, ButtonProps, alpha } from '@mui/material';
+import { alpha, Button, ButtonProps, IconButton } from '@mui/material';
 import React from 'react';
+import { useIsMobile } from '@/hooks/useIsMobile';
 
 export interface TableFilterButtonProps extends ButtonProps {
   active?: boolean;
 }
 
 const TableFilterButton: React.FC<TableFilterButtonProps> = ({
   active = false,
+  startIcon,
   ...props
 }) => {
-  return (
+  const isTiny = useIsMobile('sm');
+
+  return isTiny ? (
+    <IconButton
+      sx={(theme) => ({
+        border: `1px solid ${theme.palette.grayscale[300]}`,
```

Comment:
```suggestion
        border: `1px solid ${theme.palette.borders.dark}`,
```

For most borders throughout the app we use those colors, so lets stick to that. If its misaligned with the design language we can address during that epic